Makefile

```
@@ -18,3 +18,20 @@ ifndef VIRTUAL_ENV
 else
 	$(warning VIRTUAL_ENV variable present, already within a virtual environment?)
 endif
+
+setup: venv_check
+	python -m pip install cookiecutter
+	-rm -rf baking-test
+	cookiecutter . --no-input
+
+test: venv_check
+	make setup
```

I'd suggest adding `setup` as a goal after `venv_check` instead.

```suggestion
test: venv_check setup
```